cocos/network/WebSocket.cpp

```diff
@@ -75,6 +75,8 @@ static void _winLog(const char *format, va_list args)
             bufferSize *= 2;
 
             delete[] buf;
+            
+            return;
```

Comment:
Why to return directly here. 
The while loop will go to next step to use double size buffer for formatting the string.